Formula/saml2aws.rb

```diff
@@ -21,6 +21,19 @@ class Saml2aws < Formula
   def install
     ldflags = "-s -w -X main.Version=#{version}"
     system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/saml2aws"
+    (zsh_completion/"_saml2aws").write <<~EOS
+      #compdef saml2aws
+
+      _saml2aws_bash_autocomplete() {
+          local cur prev opts base
+          COMPREPLY=()
+          cur="${COMP_WORDS[COMP_CWORD]}"
+          opts=$( ${COMP_WORDS[0]} --completion-bash ${COMP_WORDS[@]:1:$COMP_CWORD} )
+          COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
+          return 0
+      }
+      complete -F _saml2aws_bash_autocomplete saml2aws
+    EOS
```

Comment:
```suggestion
     generate_completions_from_executable(bin/"saml2aws", "--completion-script-zsh")
```

Reply:
Please also change your commit message to `saml2aws: add completion`

Reply:
Thank you for the speedy review.

The only reason I didn't do this, is because the command also adds:
```
autoload -U compinit && compinit
autoload -U bashcompinit && bashcompinit
```

to the output. I figured this wouldn't be desirable, but I'm happy to defer to your judgement :)

Reply:
Thanks for that info. In light of that, I think it makes sense to handle it how you did.